dusty/source.py

```diff
@@ -140,3 +154,11 @@ def update_local_repo(self):
         managed_repo = git.Repo(self.managed_path)
         with git_error_handling():
             managed_repo.remote().pull('master')
+        if not self.local_is_up_to_date():
+            if force:
+                with git_error_handling():
+                    managed_repo.git.reset('--hard', 'origin/master')
+            else:
+                log_to_client('WARNING: couldn\'t update {} because of local conflicts.'.format(repo.local_path)
+                              'A container may have modified files in the repos\'s directory'
+                              'Running `dusty up` should resolve this')
```

Comment:
This is a symptom of the chronic problem that the user needs to fix. We should phrase the error message to reflect that, not give them a workaround that'll lead to them hitting the problem again and again and again.